actions/detect-secrets/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:alpine3.16
```

Comment:
Can we be a bit more specific? eg `python:3.9-alpine3.16` (see [Dockerhub](https://hub.docker.com/layers/library/python/3.9-alpine3.16/images/sha256-55bc2d88cfd5ed07d2e9420b8f83a1adc4caf49e855b5dd6437d679fae56fb3a?context=explore))